cypress/fixtures/console/application/integrations/webhook/template.json

```diff
@@ -0,0 +1,27 @@
+{
+  "templates": [{
+    "ids":{
+      "template_id":"akenza"
```

Comment:
Please avoid references to real companies/brands in within fixtures/mock data. Use e.g. `test-tamplate`, `Integrate with Test Platform`.

Reply:
I did try to change it by for reason it messed up the placeholder itself. I believe because part of the placeholder is derived from the template id. But now i did it the right way